src/Trips.js

```diff
@@ -0,0 +1,36 @@
+class Trips {
+  constructor(userTrips) {
+    this.allTrips = userTrips;
+    this.pending = [];
+    this.approved = [];
+    this.past = [];
+    this.future = [];
+  }
+
+  findTripById(id) {
+    return this.allTrips.find((trip) => {
+      return trip.id === id;
+    })
+  }
+
+  sortTripsByStatus() {
+    return this.allTrips.forEach((trip) => {
+      this[trip.status].push(trip);
+    })
+  }
+
+  sortTripsByDate() {
+    let makeDate = new Date();
+    const today = makeDate.getFullYear() + '/' + (makeDate.getMonth() + 1) + '/' + makeDate.getDate();
```

Comment:
OOOOO I like this... I think I complicated things by using dayjs and I really like seeing how to just use the Date object with < > =